configuration/configure-unhealthy-cluster-removal.adoc

```diff
@@ -0,0 +1,20 @@
+:_content-type: ASSEMBLY
+[id="configure-unhealthy-cluster-removal"]
+= Configuring Removal of Unhealthy Clusters
+include::modules/common-attributes.adoc[]
+:context: configure-unhealthy-cluster-removal
+
+toc::[]
+
+[role="_abstract"]
+{product-title} provides the ability to automatically remove unhealthy clusters in your system. If Central has been unable to reach Sensor in a cluster for a configured period of time (the default is 90 days), the cluster is considered unhealthy. Deleting unhealthy clusters allows you to monitor only active clusters. You can configure cluster removal settings by using the {product-title-short} portal or the application programming interface (API).
+
+This setting is enabled by default. To disable this setting, enter `0` in the *Cluster Decommissioning* field, as described in the following procedure. If the setting to remove unhealthy clusters is enabled, when a cluster is unreachable, the system displays a warning that the cluster is scheduled for deletion in seven days. After a cluster is removed, the removal is documented with an `info` log in the Central logs.
```

Comment:
Would "*Decommissioned cluster age*"  field instead of "*Cluster Decommissioning* field" be more accurate?

Also, in the second line, I think "when a cluster is unreachable, the cluster list page displays a warning with the number of days after which cluster will be deleted if it continues to remain unhealthy" would be better